components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -201,20 +207,27 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
         const {field, label, helpText} = this.props;
 
         let selectComponent;
-        if (field.type === AppFieldTypes.DYNAMIC_SELECT) {
+
+        switch (field.type) {
+        case AppFieldTypes.DYNAMIC_SELECT:
             selectComponent = this.renderDynamicSelect();
-        } else if (field.type === AppFieldTypes.STATIC_SELECT) {
+            break;
+        case AppFieldTypes.STATIC_SELECT:
             selectComponent = this.renderStaticSelect();
-        } else if (field.type === AppFieldTypes.USER) {
+            break;
+        case AppFieldTypes.USER:
             selectComponent = this.renderUserSelect();
-        } else if (field.type === AppFieldTypes.CHANNEL) {
+            break;
+        case AppFieldTypes.CHANNEL:
             selectComponent = this.renderChannelSelect();
-        } else {
-            return null;
+            break;
         }
 
         return (
-            <div className='form-group'>
+            <div
+                id='apps_form_select_field'
```

Comment:
Since we'll have multiple of these on the page at the same time, we should use a `class` instead of an `id`